dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -885,6 +886,15 @@ protected String getCommitID(String repositoryId, Version version) {
         return null;
     }
 
+    public String getEmail(GHMyself myself) throws IOException {
```

Comment:
Is this public for a reason?

Reply:
No reason at all, nothing outside of the class is using the getEmail() method. Is it best to change it to private?

Reply:
Changed it to private.

Reply:
Yeah, just makes things easier to reason about. 
e.g. when refactoring, I know I can change this method without breaking anything outside of this class, the IDE can tell me when the method is un-used, etc.